src/Language/PureScript/Ide/Imports.hs

```diff
@@ -220,6 +220,11 @@ addExplicitImport' decl moduleName qualifier imports =
         (insertDtor (dtor ^. ideDtorName))
         (refFromDeclaration d)
         refs
+    insertDeclIntoRefs (IdeDeclType t) refs
+      | Just _ <- find matches refs = refs
```

Comment:
well, if you used `any` instead of find you wouldn't need the pattern match on `Just`. Up to you, I don't really care.

Reply:
Knew there had to be something, hoogle skills failed me here.